src/Symfony/Bridge/Twig/TokenParser/DumpTokenParser.php

```diff
@@ -50,4 +50,9 @@ public function getTag(): string
     {
         return 'dump';
     }
+
+    private function getVarName(): string
+    {
+        return sprintf('__internal_%s', hash('sha256', uniqid(mt_rand(), true)));
```

Comment:
generating a random var name breaks reproducible template compilation, while we worked hard to make them reproducible a few years ago.